### ISSUE TITLE
sql schema version compatibility check failed

### ISSUE THREAD

**Issue by hazcod** [temporalio/temporal#896]
## Expected Behavior
Startup temporal/auto-setup in google cloud.

## Actual Behavior
```


2020-10-24 16:52:33.965 CEST2020/10/24 14:52:33 pq: relation "schema_version" already exists

2020-10-24 16:52:33.968 CEST+ temporal-sql-tool --plugin postgres --ep 10.47.192.6 -u postgres -p 5432 --db temporal_visibility update-schema -d /etc/temporal/schema/postgres/visibility/versioned

2020-10-24 16:52:34.006 CEST2020/10/24 14:52:34 UpdateSchemeTask started, config=&{DBName: TargetVersion: SchemaDir:/etc/temporal/schema/postgres/visibility/versioned IsDryRun:false}

2020-10-24 16:52:34.011 CEST2020/10/24 14:52:34 error listing schema dir:open /etc/temporal/schema/postgres/visibility/versioned: no such file or directory

2020-10-24 16:52:34.014 CEST+ '[' true '!=' true ']'

2020-10-24 16:52:34.014 CEST+ '[' false == true ']'

2020-10-24 16:52:34.014 CEST+ exec bash /start-temporal.sh

2020-10-24 16:52:34.024 CEST+ dockerize -template /etc/temporal/config/config_template.yaml:/etc/temporal/config/docker.yaml

2020-10-24 16:52:34.048 CEST+ exec temporal-server --root /etc/temporal --env docker start --services=history,matching,frontend,worker

2020-10-24 16:52:34.078 CEST2020/10/24 14:52:34 Loading config; env=docker,zone=,configDir=/etc/temporal/config

2020-10-24 16:52:34.078 CEST2020/10/24 14:52:34 Loading config files=[/etc/temporal/config/docker.yaml]

2020-10-24 16:52:34.102 CESTUnable to start server: sql schema version compatibility check failed: version mismatch for keyspace/database: "temporal". Expected version: 1.1 cannot be greater than Actual version: 0.0.
```

## Steps to Reproduce the Problem

```yaml
   spec:
      containerConcurrency: 80
      timeoutSeconds: 300
      containers:
      - image: temporalio/auto-setup@1.1.1
        ports:
        - containerPort: 7233
        env:
        - name: LOG_LEVEL
          value: INFO
        - name: AUTO_SETUP
          value: 'TRUE'
        - name: DB
          value: postgres
        - name: DB_PORT
          value: '5432'
        - name: POSTGRES_USER
          value: postgres
        - name: POSTGRES_PWD
          value: M4KaY8kpJ6fzM$2YGoiKx
        - name: POSTGRES_SEEDS
          value: 10.47.192.6
        - name: SKIP_DEFAULT_NAMESPACE_CREATION
          value: 'true'
```

## Specifications

  - Version:
  - Platform:

**Comment by wxing1292** [comment]
PostgreSQL is not officially supported.

We are doing necessary pre release tests and the official support is on the way.

The issue you are seeing is due the the move of the schema dir for better version control / support, similar to MySQL.

**Comment by wxing1292** [comment]
duplicate of #862

**Comment by wxing1292** [comment]
solved in 1.2.1 release
https://github.com/temporalio/temporal/releases/tag/v1.2.1

**Comment by jorgecuesta** [comment]
A side note for those that could came later on time here like me, check the admin-tools and server versions are on the same range, like 1.25 (at the start of the name) because that was the issue for me.
My admin-tools was on 1.24 and the server was on 1.25. The chart uses the admin-tool.image.tag because use the same docker image for the database/schema setup.
